.github/workflows/release.yml

```diff
@@ -18,7 +18,9 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: 8.0.x
+          dotnet-version: |
+            6.0.x
+            8.0.x
```

Comment:
Why 6?

Reply:
It's in the .csproj and I was trying to fix the CI. But it didn't work. I could probably just remove 6 and that would be simplest.

Reply:
Ok. Yea I think remove, since LTS support for 6 ended 2 months ago.